test/unit/matrix/test_distribution.cpp

```diff
@@ -281,3 +281,58 @@ TEST(DistributionTest, Index2DConversions) {
     }
   }
 }
+
+TEST(DistributionTest, DistanceToAdjacentTile) {
+  matrix::Distribution distr(LocalElementSize(10, 10), TileElementSize(3, 3));
+
+  // Columns
+  //
+  // Inside a tile
+  EXPECT_TRUE(distr.distanceToAdjacentTile<Coord::Col>(7) == 2);
```

Comment:
What is the reason to prefer it to 
```suggestion
  EXPECT_EQ(2, distr.distanceToAdjacentTile<Coord::Col>(7));
```
?